README.md

```diff
@@ -73,6 +73,12 @@ Valid values are `always` (slower, safest), `everysec` (happy medium), or `no` (
 
 Add extra include file paths to this list to include more/localized Redis configuration.
 
+If you want to install redis with a different package name, you need to set
```

Comment:
To follow the formatting/style of the other variables, can you put the variable name first (with four spaces instead of the backticks), then the description, and make it a full sentence, e.g. "The redis package name for installation via the system package manager. Defaults to `redis-server` on Debian and `redis` on RHEL."